Basics Practice/basics.py

```diff
@@ -0,0 +1,46 @@
+# # Python Basics
```

Comment:
I don't feel this would be necessary as these concepts are already present at https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Training%20Modules